requirements.txt

```diff
@@ -1 +1,2 @@
-dwave-ocean-sdk==1.3.0
\ No newline at end of file
+python=3.*
+dwave-system==0.8.*
```

Comment:
While I prefer `ocean-sdk`, you are correct the example does work with `dwave-system`. So 👍